quilt/tools/command.py

```diff
@@ -60,6 +59,10 @@ def _save_auth(auth):
         os.chmod(file_path, stat.S_IRUSR | stat.S_IWUSR)
         json.dump(auth, fd)
 
+def _handle_response(resp, **kwargs):
+    if resp.status_code == requests.codes.unauthorized:
+        raise CommandException("Failed to log in. Run `quilt login` again.")
```

Comment:
Since this exception can happen for reasons other than a login failure, can you change the message to be something more general, like "authentication failed?".